distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -951,6 +1002,163 @@ public void testCswCreateEventEndpoint() throws Exception {
 
     }
 
+    @Test
+    public void testDownloadFromFederatedCswSource() throws Exception {
+        RestitoRetrievalResponse restitoRetrievalResponse =
+                new RestitoRetrievalResponse("testData");
```

Comment:
We probably should use variables instead of hard-coding `"testData"` in the test methods.